setup.sh

```
@@ -0,0 +1,53 @@
+NC='\033[0m' # No Color
+
+echo -e "${GREEN}🚀 Setting up Bluelabel Agent OS development environment...${NC}"
+
+# Check if Python 3.8+ is installed
+if ! command -v python3 &> /dev/null; then
+    echo "❌ Python 3.8 or later is required. Please install it first."
+    exit 1
+fi
+
+PYTHON_VERSION=$(python3 -c "import sys; print('{}.{}'.format(sys.version_info.major, sys.version_info.minor))")
+if [[ "$PYTHON_VERSION" < "3.8" ]]; then
```

Shellcheck (0.10.0)

[error] Decimals are not supported. Either use integers only, or use bc or awk to compare.

(SC2072)